.travis.yml

```diff
@@ -21,7 +21,7 @@ env:
 addons:
   coverity_scan:
     project:
-      name: "dblock/oshi"
+      name: "oshi/oshi"
```

Comment:
Actually, our account on Coverity is [tied to the older project name](https://scan.coverity.com/projects/dblock-oshi?tab=overview).  I'd have to create a new Coverity account and lose project history to change this. 
![image](https://user-images.githubusercontent.com/9291703/58772675-e5cfac00-856e-11e9-8b9f-b6f1ac917794.png)
While there are settings to change the project coordinates (which I've done) the name can apparently not be changed.

Reply:
I undid this.